projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_connection.py

```diff
@@ -115,3 +120,8 @@ def db_connection_decorate_operation(self, decoration_cursor: DecorationCursor):
                 )
                 if self._db_default_type.lower() == "impala":
                     raise e
+
+    def db_connection_after_operation(self, execution_cursor: ExecutionCursor) -> None:
+        if self._lineage_logger:
+            impala_lineage = ImpalaLineage(self._lineage_logger)
+            impala_lineage.db_connection_after_operation(execution_cursor)
```

Comment:
Nit: It's a bit confusing having the same name for different methods in different components. I was confused which one was the override. Perhaps you can change the impala_lineage method name.